lte/gateway/fabfile.py

```diff
@@ -399,6 +400,25 @@ def bazel_integ_test_post_build(
         env.hosts = [gateway_host]
 
 
+def _setup_vm(host, name, ansible_role, ansible_file, destroy_vm, provision_vm):
+    ip = None
+    if not host:
+        host = vagrant_setup(
+            name, destroy_vm, force_provision=provision_vm,
+        )
+    else:
+        ansible_setup(host, ansible_role, ansible_file)
+        ip = host.split('@')[1].split(':')[0]
+    return host, ip
+
+
+def _setup_gateway(gateway_host, name, ansible_role, ansible_file, destroy_vm, provision_vm):
+    gateway_host, gateway_ip = _setup_vm(gateway_host, name, ansible_role, ansible_file, destroy_vm, provision_vm)
+    if gateway_ip == None:
+        gateway_ip = GATEWAY_IP_ADDRESS
```

Comment:
We should use `gateway_ip is None` here:

https://stackoverflow.com/questions/3257919/what-is-the-difference-between-is-none-and-none

Alternatively, check with if not gateway_ip, that way we also set the gateway_ip if it is an empty string.

Reply:
Done :)